src/Interactiv.js

```diff
@@ -0,0 +1,53 @@
+class Interactive {
+  static checkboxevent() {
+    const checkboxes = document.querySelectorAll('input[type=checkbox][name=checking]');
+    let enabledSettings = [];
+    checkboxes.forEach((checkbox) => {
+      checkbox.addEventListener('change', () => {
+        enabledSettings = Array.from(checkboxes).filter((i) => i.checked).map((i) => i.value);
+        const hidden = document.querySelector('.hidden');
+        hidden.value = enabledSettings;
+      });
+    });
+  }
+
+  static clearcomplt() {
+    let mylocaldata = JSON.parse(localStorage.getItem('toDoList'));
+    if (mylocaldata === null) { mylocaldata = []; }
+    const completearry = [];
+    mylocaldata.forEach((element, i) => {
+      if (mylocaldata[i].complete === true) {
+        completearry.push(mylocaldata[i]);
+        mylocaldata.splice(i, 1);
+      }
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly when I try to tick the tasks, this part cannot be set properties of null. Please fix it 🙏🏼 

![image](https://github.com/gergitokusse/To-Do-list/assets/63915024/b4ea8870-6084-4b72-9dda-7397dc420429)

Reply:
Well resolved ✔️